### PR TITLE
fix(ssr): fix refresh in SSR mode not keeping the server list and current server

### PR DESCRIPTION
**Pull request by MrTimscampi**
This moves the persistence of the `server` Vuex module to use cookies (Via vuex-persistedstate).

https://user-images.githubusercontent.com/19396809/106182473-239b3280-619f-11eb-9f0b-04b21820c61d.mp4

As a result, some errors about the Swipers showed up, so I set these components to be client-only.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,6 +80,8 @@
     "@nuxtjs/imagemin": "^0.2.0",
     "@nuxtjs/stylelint-module": "^4.0.0",
     "@nuxtjs/vuetify": "^1.11.3",
+    "@types/cookie": "^0.4.0",
```
Comment (ferferga) [inline package.json:83]: Those shouldn't be added to tsconfig as well?

Comment (MrTimscampi) [reply package.json:83]: It'll detect everything in @types automatically.

Comment (MrTimscampi) [reply package.json:83]: Precision: when importing the package as modules.

If it's global stuff you're never importing (Like MediaSessions) then you need it in tsconfig.

**Review by ferferga — APPROVED** [review]
LGTM

**Comment by Camc314** [comment]
This seems to have broken servers in static mode

**Pull request merged**